app/src/main/java/com/example/runningavater/initialFlow/InitialFlow10.kt

```diff
@@ -1,10 +1,93 @@
 package com.example.runningavater.initialFlow
 
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.rememberNavController
+import com.example.runningavater.R
 import com.example.runningavater.initialFlow.components.InitialFlowBackground
+import com.example.runningavater.initialFlow.components.NextButton
+import com.example.runningavater.ui.theme.RunningAvaterTheme
 
 @Composable
-fun InitialFlow10Screen() {
+fun InitialFlow10Screen(navController: NavHostController) {
     InitialFlowBackground {
+        Box(modifier = Modifier.fillMaxSize()) {
+            Column(
+                modifier =
+                    Modifier
+                        .fillMaxWidth()
+                        .align(Alignment.TopCenter)
+                        .padding(0.dp, 50.dp, 0.dp, 0.dp),
+                horizontalAlignment = Alignment.CenterHorizontally,
+            ) {
+                Text(
+                    text = "さっそくダイエットを",
+                    color = Color.Black,
+                    fontSize = 32.sp,
+                    textAlign = TextAlign.Center,
+                )
+                Text(
+                    text = "始めよう！",
+                    color = Color.Black,
+                    fontSize = 32.sp,
+                    textAlign = TextAlign.Center,
+                )
```

Comment:
[must]
横幅が狭い端末で開業したときに行間が気になるので、きちんとline heightを設定するべき
ついでにTextも1つにまとめれそう

```suggestion
                Text(
                    text = "さっそくダイエットを\n始めよう！",
                    color = Color.Black,
                    fontSize = 32.sp,
                    textAlign = TextAlign.Center,
                    lineHeight = 46.3.sp,
                )
```

Reply:
[add38aa](https://github.com/NID-kt/toryumon-team-f/pull/107/commits/add38aa826ca33e9ad438bebdf9c4dd889d9ff4e) で修正